meta2v2/meta2_backend.c

```diff
@@ -1298,11 +1356,37 @@ meta2_backend_insert_beans(struct meta2_backend_s *m2b,
 			else
 				err = _db_insert_beans_list (sq3->db, beans);
 			if (!err) {
+				gint64 damaged_objects = m2db_get_damaged_objects(sq3);
 				gint64 missing_chunks = m2db_get_missing_chunks(sq3);
+
+				GHashTable *content_ids = g_hash_table_new_full(
+						g_str_hash, g_str_equal, NULL, NULL);
```

Comment:
The key free function (3rd parameter) should be set to `g_free`. See comment below.